lego/apps/users/constants.py

```diff
@@ -80,10 +80,10 @@ class FSGroup(Enum):
     def values(cls) -> list[str]:
         return [e.value for e in cls]
 
-    MTDT = "fc:fs:fs:prg:ntnu.no:mtdt"
-    MTKOM = "fc:fs:fs:prg:ntnu.no:mtkom"
-    MIDT = "fc:fs:fs:prg:ntnu.no:midt"
-    MSTCNNS = "fc:fs:fs:prg:ntnu.no:mstcnns"
+    MTDT = "fc:fs:fs:prg:ntnu.no:MTDT"
+    MTKOM = "fc:fs:fs:prg:ntnu.no:MTKOM"
+    MIDT = "fc:fs:fs:prg:ntnu.no:MIDT"
+    MSTCNNS = "fc:fs:fs:prg:ntnu.no:MSTCNNS"
 
```

Comment:
As long as we are 100% certain that these are correctly formatted :eyes:

Reply:
Time will tell